### DISCUSSION TITLE
求打包最新的whatsapp

### DISCUSSION THREAD

**Discussion by ahtoh84** [tw93/Pake#196] [Q&A]
官方的Mac版十分的卡顿，适配M芯片的还没发布。项目里之前打包的那个已无法使用，还请帮忙打包个最新版的，非常感谢！

**Comment by cnmeeia** [comment]
https://github.com/cnmeeia/pake.macapp/releases/download/V1.3.0/whatsapp.dmg

**Reply by tw93** [reply]
感谢感谢

**Reply by cnmeeia** [reply]
不用这么客气啦☺️

**Comment by ahtoh84** [comment]
非常感谢！！
